### ISSUE TITLE
Fix nightly update and database refresh tasks

### ISSUE THREAD

**Issue by ccostino** [18F/openFEC#2196]
We've recently run into a few issues with our nightly update tasks as well as the full-fledged refresh tasks and the underlying SQL scripts that power them, largely due to table name changes but also because of a few other reasons.

First, a few tables need to have their names adjusted and have `vw` attached to the end:

* `fec_vsum_sched_a`
* `fec_vsum_sched_b`
* Possibly a few others related to `fec_vsum_*` tables

Second, the nightly updates are running into two issues.  In `dev`, it's due to a primary key violation:

```
2017-02-08 09:00:00 ERROR:manager:(psycopg2.IntegrityError) duplicate key value violates unique constraint "ofec_sched_e_tmp_pkey"
DETAIL:  Key (sub_id)=(4062120141215616440) already exists.
```

In `stage` and `prod`, it's due to missing tables:

```
2017-02-08 09:00:00 ERROR:manager:(psycopg2.ProgrammingError) relation "fec_vsum_f57_queue_old" does not exist
```

Upon looking, the `fec_vsum_f57_queue_new` table doesn't exist either, but both tables are present in `dev`.

I haven't been able to tell if there are any other underlying issues at the moment, but that's the start of them.  We need to rectify these as soon as we can so we can get the nightly refreshes working again and run `update_all` on `dev`, so I'm placing it in this sprint as a bug and something to tackle before the next release.

**Comment by LindsayYoung** [comment]
if you don't see fec_vsum_f57_queue_new probably need to run prepare schedule e

**Comment by jontours** [comment]
Yeah running `prepare_schedule_e` should fix the issue with the missing table.  The duplicate key issue may be harder to find.

**Comment by ccostino** [comment]
Okay great, thank you both!  I'll give that a shot - is it safe to run `prepare_schedule_e` in production during an off peak time to get that addressed?

Also flagging #2197 here as I suspect this data isn't showing up due to these tasks not running.

**Comment by ccostino** [comment]
I just ran `prepare_schedule_e` in `stage` and `prod` after testing things out with locally to be sure.  Indexes appear to be in tact across the board for the itemized data and the tables that weren't in `stage` or `prod` when this started happening are back.  I will check in the morning to see if the nightly refresh worked or if another issue was lurking underneath.

**Comment by ccostino** [comment]
Okay, we've had some progress, but unfortunately this exposed another issue:  `prod` and `stage` are now throwing this error:

```
2017-02-09 09:38:49 ERROR:manager:(psycopg2.ProgrammingError) relation "ao" does not exist
```

@anthonygarvan, @vrajmohan, or @jontours:  Do any of you know what needs to be run to add this missing relation?

**Comment by vrajmohan** [comment]
I would guess that `index_advisory_opinions` failed at https://github.com/18F/openFEC/blob/develop/webservices/legal_docs/load_legal_docs.py#L135.

**Comment by vrajmohan** [comment]
I was able to confirm that the lack of schema qualification is indeed the issue. Submitting PR.

**Comment by anthonygarvan** [comment]
Crap! Yes, the schema needs to be specified as aouser.ao. It also needs to
be changed in 2-3 more places. I have no idea why this is failing now but
was working for many months. See PR:
https://github.com/18F/openFEC/pull/2195

**Comment by anthonygarvan** [comment]
Already have a PR (jinx!). Please review it ^^

**Comment by ccostino** [comment]
Thank you both!

**Comment by ccostino** [comment]
We're currently looking into the last bits of this to get ourselves back on track.  @jontours and @LindsayYoung have been helping me (thank you both!).

**Comment by noahmanger** [comment]
Resolved by #2218
